src/main/Resources/Messages_fr.properties

```diff
@@ -0,0 +1,26 @@
+next=Suivant
+close=Fermer
+redo=REFAIRE
+welcome=Content de te Revoir
+create=CRÉER
+languageSelectorHint = Try English or ENG
+targetLanguages = Langue Objectif
+sourceLanguages = Lanuge Source
```

Comment:
spelling?

Reply:
that's french?

Reply:
`Langue` on line 7 vs `Lanuge` on line 8?